nixos/modules/services/networking/nats.nix

```diff
@@ -0,0 +1,174 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.nats;
+
+  confFile = pkgs.writeText "nats.conf" (builtins.toJSON cfg.settings);
+
+in {
+
+  ### Interface
+
+  options = {
+    services.nats = {
+      enable = mkEnableOption "NATS messaging system";
+
+      package = mkOption {
```

Comment:
No, there isn't. Personally I always like the availability of a `package` option so i'm able to pin a specific package version. But now that I write this, just realized that this can be done via an overlay as well :sweat_smile:

Reply:
Exactly :+1: